### ISSUE TITLE
Game crashes after changing settings.

### ISSUE THREAD

**Issue by ghost** [lklacar/openjk-emscripten#2]
When a game setting is changed that requires a restart, the game crashes.

**Comment by lklacar** [comment]
Yes, I am aware of this issue. Honestly, I'm not really sure why this is happening. I'm assuming this is because the original engine uses quite a lot of dynamic linking, and this project is completely statically linked. This is going to be a pain to debug and fix, and its quite a low priority for now. 

If you're willing, feel free to take a look yourself. I'd be happy to take a look at the PR and accept it if it fixes the issue. Sorry if I couldn't be of more help.

**Comment by ghost** [comment]
It's probably just Emscripten, if the game restarts it probably thinks it's crashed, seeing as not many games, at least that I've played require the whole game to restart after changing a setting.
